src/coreclr/debug/daccess/enummem.cpp

```diff
@@ -25,6 +25,8 @@
 #include <interoplibabi.h>
 #endif // FEATURE_COMWRAPPERS
 
+#include "cdacplatformmetadata.hpp"
```

Comment:
Is this needed? There is no other change in this file, so I guess it is a remainder of a previous state of this change.

Reply:
There is a change to `dacvars.h` that requires this.